docker/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo Building container
+docker build -t gobuilder "$(pwd)"
+echo Running make script
+docker run --rm -v "/$(pwd)/.."://usr/src/myapp -w //usr/src/myapp gobuilder make
```

Comment:
Can we please mount to `/go/src/github.com/kris-nova/kubicorn` and have `$GOPATH=/go`?